### ISSUE TITLE
🐛 Bug: MSSQL Decimal Field is not being set with Precision

### ISSUE THREAD

**Issue by lvarming** [nocodb/nocodb#4850]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

1. Create New Field of Type Decimal

![Screenshot 2023-01-16 073648](https://user-images.githubusercontent.com/111744156/212613528-025e9268-6640-42d8-9f01-d0cf86c546ea.png)

### Desired Behavior

Given that Decimal typically is stored in decimal with 2 decimal places. The Data Type should be something like Decimal(18,4). However, the system is placing the Decimal Type as Decimal (18,0) with no way of changing it.

![Screenshot 2023-01-16 073840](https://user-images.githubusercontent.com/111744156/212613755-1e2859f1-09a4-4c0a-8c53-2e2cc93b9583.png)

### Project Details

This problem is similar to [issue 4833](https://github.com/nocodb/nocodb/issues/4833)

NocoDB: Docker, 0.101.0
Database used: Tested with both MS SQL Server 2019 Express and MS SQL Server 2017

**Comment by chxp** [comment]
Node: **v16.17.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
RootDB: **mysql2**
PackageVersion: **0.104.3**

It seems the decimal-type value has no precision choice in this version, also. 

Only the choice of 'default value' displays in the dialog when creating a decimal column.

 
![Screenshot_20230208_153736](https://user-images.githubusercontent.com/14882692/217468235-e6577971-8c21-4bf9-bfcd-7a02085d2fe4.png)


Thus, 2.1212 will display as 2.12.

**Comment by lvarming** [comment]
Double-click hide-all and it will open the "advanced" options.

**Comment by chxp** [comment]
I have double-click it and the picture is the results after I click the "advanced" options.

**Comment by chxp** [comment]
No matter how many times I click the button, there are only two situations: no "default value" choice and only "default value choice"

**Comment by chxp** [comment]
Yes, I now found that double-clicking the button will enable advanced options such as precision.

**Comment by dstala** [comment]
We have not noticed much traction with usage of MS SQL Server support & hence we decided to narrow down support mainly to MySQL & PG. Please feel free to connect with us to discuss more on your use-case [here](https://cal.com/nocodb).
